cloud/shared/variable_definitions.json

```diff
@@ -97,5 +97,11 @@
     "secret": false,
     "tfvar": false,
     "type": "string"
+  },
+  "FEATURE_FLAG_STATUS_TRACKING_ENABLED": {
+    "required": false,
+    "secret": false,
+    "tfvar": false,
```

Comment:
That should be true given that you pass it to the terraform.

Reply:
fixed.